sigwinch.go

```diff
@@ -0,0 +1,44 @@
+package tcell
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+type sigwinch struct {
+	ch chan os.Signal
+
+	// true if the channel is reporting SIGWINCH's from a tty other than the current one
```

Comment:
@dbentley sooooo this was my attempt to solve the fact that we want to call notify/stop independently of `NewTerminfoScreen...` and want to be able to do it multiple times... but can't notify/stop/notify/stop if the channel is broadcasting signals from some remote terminal. 😬 What do you think?